AI/VCAI/AIhelper.h

```diff
@@ -16,19 +16,21 @@
 
 #include "ResourceManager.h"
 #include "BuildingManager.h"
+#include "Pathfinding/CPathfindingManager.h"
 
 class ResourceManager;
 class BuildingManager;
 
 
 //indirection interface for various modules
-class DLL_EXPORT AIhelper : public IResourceManager, public IBuildingManager
+class DLL_EXPORT AIhelper : public IResourceManager, public IBuildingManager, public IPathfindingManager
```

Comment:
What is point of this inheritance? If you will delete it, it will be work as well.

Reply:
Not mine. //indirection interface for various modules
I just did the same way as it was already done

Reply:
huh ok, can stay for now, let's hope that someone will change it in seperate PR